common/app/services/DynamoDB.scala

```diff
@@ -8,8 +8,8 @@ import conf.Configuration
 
 
 object DynamoDB {
-  private lazy val credentials = Configuration.aws.mandatoryCredentials
-  private lazy val region = Region.getRegion(Regions.fromName(Configuration.aws.region))
+  private val credentials = Configuration.aws.mandatoryCredentials
```

Comment:
You made all the properties lazy, except these two where you removed the laziness, is that voluntary?